todo_app/app.py

```diff
@@ -1,18 +1,42 @@
 from flask import Flask, render_template, request, redirect, url_for
-
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import get_items, add_item, get_list_id, change_status
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
+
 @app.route('/')
 def index():
     items = get_items()
     return render_template('index.html', items=items)
 
+
 @app.route('/items/new', methods=['POST'])
 def new_item():
     title = request.form.get('item_title')
-    add_item(title)
-    return redirect(url_for('index'))
\ No newline at end of file
+    status = request.form.get('status')
+    list_id = get_list_id(status)
+    add_item(list_id, title)
+    return redirect(url_for('index'))
+
+
+@app.route('/<item_id>/done', methods=['POST', 'PUT', 'GET'])
+def move_to_done(item_id):
+    list_id = get_list_id('Done')
+    change_status(item_id, list_id)
+    return redirect(url_for('index'))
+
+
+@app.route('/<item_id>/in_progress', methods=['POST', 'PUT', 'GET'])
```

Comment:
It looks like you only expect traffic to POST to this endpoint, is there any reason to enable PUT and GET in addition? It can feel like you're just adding in helpful flexibility, but often it's better to get a clear signal that says "someone's doing something unexpected" than to try account for each variation, particularly since those variations will inevitably end up being untested since you're not expecting anyone to ever use them!

Reply:
Noted, I have removed the PUT and GET methods